app/controllers/surveys_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class SurveysController < ApplicationController
+  before_action :verify_current_user
+  before_action :init_graphql_client
+
+  def index
+    results = @smart_village.query <<~GRAPHQL
+      query {
+        surveys {
+          id
+          title
+          questionTitle
+          visible
```

Comment:
what is visible for?

Reply:
"Sichtbarkeit" in the table. this way data can be toggled based on users.
restricted users for example can create surveys, which are invisible by default. a more powered user need to set visibility for being displayed for everybody ("freigeben").

the logic is the same as for existing resources.

Reply:
and the query from the app automatically only gets `visible` surveys?

Reply:
yes. should be like that already, if not, it will be later.

Reply:
:+1: